blocks/header/header.js

```diff
@@ -355,13 +355,70 @@ function settingAltTextForSearchIcon() {
   searchImage.setAttribute('title', listOfAllPlaceholdersData.searchAltText);
 }
 
-async function fetchingPlaceholdersData() {
+async function fetchingPlaceholdersData(placeholdersData) {
   listOfAllPlaceholdersData = await fetchLanguagePlaceholders();
   const hamburger = document.querySelector('.nav-hamburger');
-  hamburger.setAttribute('title', listOfAllPlaceholdersData.hamburgerAltText);
+  hamburger.setAttribute('title', placeholdersData.hamburgerAltText);
   makeImageClickableNSettingAltText();
   settingAltTextForSearchIcon();
 }
+const toggleExpandLanguageSelector = (e) => {
+  const toggleContainer = e.currentTarget;
+  if (toggleContainer) {
```

Comment:
Instead you can use `if (!toggleContainer) return;` so that you don't need to add a large chunk of JS inside `if`